JapaneseFilter/sections/css_extended.txt

```diff
@@ -27,5 +27,6 @@ mindhack2ch.com##.article-body > .rss:has(> script)
 mindhack2ch.com##.article-post-bottom > .rss-article:has(> .rss-article-item)
 !
 momon-ga.com##.content > .section-box[-ext-has='a[target="_blank"]']
+iza.ne.jp#$#.sideFixedWrapper { position: relative!important; }
```

Comment:
Not css_extended. It is general_extensions.
I also don't see a difference with this rule.